administrate.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.add_dependency "bourbon", "~> 5.0.0.beta.7"
   s.add_dependency "datetime_picker_rails", "~> 0.0.7"
   s.add_dependency "jquery-rails", "~> 4.0"
-  s.add_dependency "kaminari", "~> 0.16"
+  s.add_dependency "kaminari", "~> 1.0"
```

Comment:
Let's go for it and see what ends up happening!

Kaminari is pretty well used and stable, so my feeling is that it's best for us to be more liberal with our versioned dependencies.